subprojects/packagefiles/glfw/meson_options.txt

```diff
@@ -0,0 +1,26 @@
+#####  GLFW build options
+
+#@ Build options
+option('build-docs',
+        type : 'boolean',
+        value : false,
+        description : 'Build GLFW documentation?')
```

Comment:
See above on why I think this option is not necessary.

Reply:
> What's the point of having a build option for enabling a feature that is not implemented?

It provides a framework for those that follow, the same as the `win32` and `cocoa` display API options. Should those be removed? Is it _necessary_? No, it is not. Is it _useful_? I think so.